Susi/Model/Client/Convenience.swift

```diff
@@ -535,6 +535,7 @@ extension Client {
                 completion(false, ResponseMessages.ServerError)
             } else if results != nil {
                 completion(true, nil)
+                return
```

Comment:
Why additional return's?

Reply:
Otherwise, it returns always false from outer completion: `completion(false, ResponseMessages.ServerError)`